prometheus.lua

```diff
@@ -261,6 +261,32 @@ local function construct_bucket_format(buckets)
   return "%0" .. (max_order + max_precision + 1) .. "." .. max_precision .. "f"
 end
 
+-- Format bucket format when expose metrics.
+--
+-- This receives a key, remove leading and trailing zeros and return the cleaner
+-- number, in order to make the output more clear, without effect on the increasing
+-- order.
+--
+-- Args:
+--   key: the metric key
+--
+-- Returns:
+--   (string) the formatted key
+local function format_bucket_when_expose(key)
+  local part1, bucket, part2 = key:match('(,le=")(.*)(")')
+  if part1 == nil then
+    part1, bucket, part2 = key:match('({le=")(.*)(")')
+  end
+  local all_bucket_string = table.concat({part1, bucket, part2}, "")
+  if bucket == "Inf" then
+    return key
```

Comment:
Perhaps this function should also replace "Inf" with "+Inf", which would allow us to remove lines 836-839.

Reply:
Cool! Fixed